DataFormats/Detectors/EMCAL/include/DataFormatsEMCAL/EMCALBlockHeader.h

```diff
@@ -0,0 +1,36 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef O2_EMCALBLOCKHEADER
+#define O2_EMCALBLOCKHEADER
+
+#include "Headers/DataHeader.h"
+
+namespace o2
+{
+
+namespace emcal
+{
+
+/// @struct EMCALBlockHeader
+/// @brief Header for EMCAL flagging the following EMCAL payload
+struct EMCALBlockHeader : public o2::header::BaseHeader {
+  // Required to do the lookup
+  constexpr static const o2::header::HeaderType sHeaderType = "EMCBlkH";
```

Comment:
since there is the cxx file with the static variable you should do the assignment there

Reply:
Done

Reply:
Leading to a compiler error under gcc 7.3 (not under clang on mac):

mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/build_o2checkcode_o2/0/DataFormats/Detectors/EMCAL/include/DataFormatsEMCAL/EMCALBlockHeader.h:25:49: error: constexpr static data member 'sHeaderType' must have an initializer
   constexpr static const o2::header::HeaderType sHeaderType;
                                                 ^~~~~~~~~~~